eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -276,60 +255,60 @@
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-keyvault-cryptography</artifactId>
-          <version>${azure-keyvault.track-one.version}</version>
+          <version>1.2.3</version> <!-- {x-version-update;com.microsoft.azure:azure-keyvault-cryptography;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-keyvault-core</artifactId>
-          <version>${azure-keyvault.track-one.version}</version>
+          <version>1.2.3</version> <!-- {x-version-update;com.microsoft.azure:azure-keyvault-core;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-keyvault-extensions</artifactId>
-          <version>${azure-keyvault.track-one.version}</version>
+          <version>1.2.3</version> <!-- {x-version-update;com.microsoft.azure:azure-keyvault-extensions;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-keyvault-webkey</artifactId>
-          <version>${azure-keyvault.track-one.version}</version>
+          <version>1.2.3</version> <!-- {x-version-update;com.microsoft.azure:azure-keyvault-webkey;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-batch</artifactId>
-          <version>${azure-batch.version}</version>
+          <version>7.1.0-preview.1</version> <!-- {x-version-update;com.microsoft.azure:azure-batch;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-eventhubs</artifactId>
-          <version>${azure-eventhubs.version}</version>
+          <version>3.1.0</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-eventhubs-eph</artifactId>
-          <version>${azure-eventhubs-eph.version}</version>
+          <version>3.1.0</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
         </dependency>
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-eventhubs-extensions</artifactId>
-          <version>${azure-eventhubs.version}</version>
+          <version>3.1.0</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-extensions;current} -->
         </dependency>
         <!-- Storage is excluded from Spotbugs report as it collides with the storage library
              dependency in eventhubs code. This issue is under investigation.
               Issue Link: https://github.com/Azure/azure-sdk-for-java/issues/3096 -->
         <!--dependency>
```

Comment:
Right here. 

It looks like this:

```
<!--dependency>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>azure-storage-blob</artifactId>
    <version>${azure-storage-blob.version}</version>
    <version>11.0.2</version> <!-- {x-version-update;com.microsoft.azure:azure-storage-blob;current} -->
</dependency-->
```
When it should look something like this
```
<!--dependency>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>azure-storage-blob</artifactId>
    <version>11.0.2</version>  {x-version-update;com.microsoft.azure:azure-storage-blob;current}
</dependency-->
```
Nested comments end with the first `-->`, not the second. That's why it was flipping out.

Reply:
I am working on another version of doc-warden that will add a bunch of verbosity to the output.

Reply:
Ah, so it's the nested comments. Thanks for taking a look at this @scbedd